Spokestack/TextToSpeechInput.swift

```diff
@@ -8,46 +8,47 @@
 
 import Foundation
 
+/// Designate the format the input is provided in.
+@objc public enum TTSInputFormat: Int {
+    @available(*, unavailable, message: "Text-only input is not currently supported. Encase your text with <speak>text</speak> for SSML compatibility.")
```

Comment:
I think we'll have API support for plaintext soon enough that we can reverse this deprecation and not merge until it exists.